src/olympia/stats/models.py

```diff
@@ -19,34 +19,31 @@ class StatsSearchMixin(SearchMixin):
 
 class DownloadCount(StatsSearchMixin, models.Model):
     id = PositiveAutoField(primary_key=True)
-    # has an index `addon_id` on this column...
     addon = models.ForeignKey('addons.Addon', on_delete=models.CASCADE)
 
-    # has an index named `count` in dev, stage and prod
-    count = models.PositiveIntegerField(db_index=True)
+    count = models.PositiveIntegerField()
     date = models.DateField()
     sources = JSONField(db_column='src', null=True)
 
     class Meta:
         db_table = 'download_counts'
-
-        # additional indices on this table (in dev, stage and prod):
-        # * KEY `addon_and_count` (`addon_id`,`count`)
-        # * KEY `addon_date_idx` (`addon_id`,`date`)
-
-        # in our (dev, stage and prod) database:
-        # UNIQUE KEY `date_2` (`date`,`addon_id`)
-        unique_together = ('date', 'addon')
+        indexes = [
+            # FIXME: some of these might redundant.
```

Comment:
Given that it's been an ongoing investigation, there's https://github.com/mozilla/addons-server/issues/5712 that covers that part. Maybe worth linking to it?

Reply:
1163b56517f2961ae5f136dc6d937d276fcfc897